src/test/ui/panic-runtime/abort-link-to-unwind-msvc-no-std-link-to-libcmt.rs

```diff
@@ -0,0 +1,32 @@
+// build-pass
+// compile-flags: -C panic=abort -C target-feature=+crt-static
+// aux-build:exit-success-if-unwind-msvc-no-std.rs
+// only-msvc
+// Test that `no_std` with `panic=abort` under MSVC toolchain
+// doesn't cause error when linking to libcmt.
+// We don't run this executable because it will hang in `rust_begin_unwind`
```

Comment:
Do we expect that this should actually run successfully? If the MSCRT is linked in statically, then we should be able to handle linking another crate which is compiled with `-Cpanic=unwind`, no?

Reply:
At first, I expect it fail because `panic!` is in a `panic=abort` crate, and unwind won't occur; but it hangs in `rust_begin_unwind`.

Reply:
I solved this problem now. It should be able to run and fail.

Reply:
This comment should be changed then.